easybuild/easyconfigs/g/Gaussian/Gaussian-16.C.02-AVX2.eb

```
@@ -0,0 +1,65 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
+
+name = 'Gaussian'
+version = '16.C.02'
+versionsuffix = '-AVX2'
+local_developmentid = 'E6B-432N'
+
+homepage = 'https://www.gaussian.com/'
+
+# For the sake of users I advise changing the last line in the description
+# and versionsuffix to match the actual installed version.
+description = """
+Gaussian provides state-of-the-art capabilities for electronic structure
+modeling. Gaussian 16 is licensed for a wide variety of computer
+systems. All versions of Gaussian 16 contain every scientific/modeling
+feature, and none imposes any artificial limitations on calculations
+other than your computing resources and patience.
+
+This is the official gaussian AVX2 build.
+"""
+
+toolchain = SYSTEM
+
+# Gaussian is licensed software.
+# You need to pick the tar file of your choice from the DVD,
+# put it in your sources tree with a proper name.
+# (should be a tree that ordinary users can't read due to license restrictions)
+# and change the name in "sources"
+sources = ['%(version)s%(versionsuffix)s-'+local_developmentid+'.tbJ']
```

we can list the sources in a slightly different way so that we can specify the `extract_cmd` used to extract the sources:
```suggestion
sources = [{
    'filename': '%%(version)s%%(versionsuffix)s-%s.tbJ' % local_developmentid,
    'extract_cmd': 'tar xvfJ %s',
}]
```

---

Excellent. Thanks for that edit!